monasca_persister/tests/test_events.py

```diff
@@ -36,9 +36,9 @@ def tearDown(self):
 
     def test_parse_event(self):
         event = self._load_event('event_1')
-        (project_id, timestamp, event_type,
```

Comment:
Nit: why are we renaming this? project_id is the modern terminology

Reply:
This is from the cherry-pick, not something that I put in.

Reply:
Is your cherry-pick out of date?
https://review.opendev.org/#/c/689513/5/monasca_persister/tests/test_events.py

Reply:
Ah okay looks like in the merge conflict, the name in the HEAD was tenant_id so I changed everything to match that... this shouldn't affect the actual functionality though. I'd rather not redeploy everything just to incorporate this change so lets merge this?